functions-and-scope.js

```diff
@@ -40,6 +70,16 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat ik alle waardes uit de array kan langslopen, ook als de array wel 100 entries zou bevatten?
 // Log het antwoord in de terminal.
 
+let counter = 0;
+let totalGrades = 0;
+for (let i = 0; i < grades.length; i++) {
+    totalGrades += grades[i];
+    counter++;
+}
+const averageGrade1 = totalGrades / counter;
```

Comment:
Je hoeft niet een losse counter bij te houden, omdat die altijd gelijk is aan `grades.length`
```suggestion
const averageGrade1 = totalGrades / grades.length;
```